src/libraries/System.Drawing.Common/tests/FontTests.cs

```diff
@@ -978,5 +978,18 @@ private static void VerifyFont(Font font, string expectedName, float expectedEmS
             Assert.False(font.IsSystemFont);
             Assert.Empty(font.SystemFontName);
         }
+
+        [ConditionalFact(Helpers.IsDrawingSupported)]
+        public void GetHashCode_DifferentNameSameSizeStyleUnit_HashCodeIsNotSame()
+        {
+            using (FontFamily family1 = FontFamily.GenericMonospace)
```

Comment:
It seems like RedHat and Fedora are returning the same font family for `GenericMonospace` and `GenericSericf` and that's why the test is failing on those configurations, can we try and use another font family?